external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -269,7 +276,7 @@ pub enum FieldBindings {
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum LValue_ {
-    Var(Mutability, ModuleAccess, Option<Vec<Type>>),
+    Var(Option<Mutability>, ModuleAccess, Option<Vec<Type>>),
```

Comment:
When would this be `None`?

Reply:
This is None because we use Lvalue for assignment as well as bindings. There is no `mut` in the assignment case, and that is guaranteed by the grammar